source/android/mobile/src/main/java/io/adaptivecards/adaptivecardssample/CustomObjects/Actions/CustomActionListener.java

```diff
@@ -9,6 +9,7 @@
 import io.adaptivecards.objectmodel.BaseActionElement;
```

Comment:
I think you may be missing the update to the sample applications or a way to exercise this API manually

Reply:
The sample app will use this API. The change is present in the 3rd PR for the same feature